File: TrabajoI.C/C#/clase/ClaseB.cs
public class B : Figura
{
    private int valor;

    public B(int valor)
    {
        this.valor = valor;
        Console.WriteLine($"Constructor con parámetros en B: {valor}");
    }

    public override void Dibujar()
    {
        Console.WriteLine("Dibujando B");
    }
}

File: TrabajoI.C/Java/interfaz/ClaseB.java
package interfaz;

public class ClaseB implements Operable {
    int valor;

    public ClaseB(int valor) {  // Constructor con parámetros
        this.valor = valor;
        System.out.println("Constructor con parámetros en B: " + valor);
    }

    @Override
    public void operar() {
        System.out.println("Operando en B");
    }
}
